code/__DEFINES/research.dm

```diff
@@ -74,3 +74,5 @@
 #define SCIPAPER_COOPERATION_INDEX 1
 #define SCIPAPER_FUNDING_INDEX 2
 #define SCIENTIFIC_COOPERATION_PURCHASE_MULTIPLIER 0.01
+/// How much money is one point of gain worth.
+#define SCIPAPER_GAIN_TO_MONEY 125
```

Comment:
instead of trying to fuck about with a mulitipiler for the next 500 years

Just pick the top and bottom of the range you want papers to earn and then scale the resulting science "value" of papers in between them with a normalising function https://stackoverflow.com/questions/5294955/how-to-scale-down-a-range-of-numbers-with-a-known-min-and-max-value

There's one buried in the math files somewhere tbh

Reply:
that way when people fuck about with the underlying science points formula your shit doesn't get wacked.

Reply:
Already have something similar in https://github.com/tgstation/tgstation/blob/6ed3637b62f639b5e7bcd772988755dce4754755/code/modules/research/ordnance/_scipaper.dm#L43. Max value is just twice of whatever the gain number is. It's just the multiplier that was out of wack. Hope this is adequate.